rs/nns/governance/src/neuron/types.rs

```diff
@@ -942,6 +951,8 @@ impl TryFrom<NeuronProto> for Neuron {
             dissolve_state,
             aging_since_timestamp_seconds,
         })?;
+        let visibility =
+            visibility.and_then(|visibility| crate::pb::v1::Visibility::try_from(visibility).ok());
```

Comment:
If the conversion is unsuccessful, should we log or propagate the error instead of hiding it? NCR

Reply:
Done. Propagate.